src/ray/core_worker/test/direct_task_transport_test.cc

```diff
@@ -92,8 +91,7 @@ TEST(TestMemoryStore, TestPromoteToPlasma) {
 }
 
 TEST(LocalDependencyResolverTest, TestNoDependencies) {
-  auto ptr = std::shared_ptr<CoreWorkerMemoryStore>(new CoreWorkerMemoryStore());
-  auto store = std::make_shared<CoreWorkerMemoryStoreProvider>(ptr);
+  auto store = std::shared_ptr<CoreWorkerMemoryStore>(new CoreWorkerMemoryStore());
```

Comment:
Now would also be a nice time to put the common setup code in a fixture :)

Reply:
I have a personal bias against fixtures (I think some level of redundancy is fine for test code, and you should prefer to use "normal" helper methods and so on instead of special test tools when possible), but maybe we can do it in another pr.

Reply:
Hmm I do think it's nice to reduce the test cases a bit, but I don't feel strongly about it.

Reply:
Yeah, it is getting a bit long here.